inference-engine/src/mkldnn_plugin/CMakeLists.txt

```diff
@@ -45,10 +45,10 @@ if(SELECTIVE_BUILD STREQUAL "ON")
     endif()
 endif()
 
-target_link_libraries(${TARGET_NAME} PRIVATE mkldnn
-                                             inference_engine
+target_link_libraries(${TARGET_NAME} PRIVATE mkldnn inference_engine inference_engine_legacy
                                              inference_engine_transformations
```

Comment:
CPU plugin does not depend on `inference_engine_legacy` anymore

Reply:
done